src/Feature/Account/tests/Controllers/AccountsControllerTests.cs

```diff
@@ -85,7 +85,7 @@ public void CreateAccount_ShouldCallExecuteMethod()
         public void GetAddressList_ShouldCallExecuteMethod()
         {
             // act
-            this.controller.GetAddress();
+            this.controller.GetAddresses();
```

Comment:
did you removed routeCollection.MapRoute?

Reply:
Old order controller was api so is has not map route.
This specific because I forgot do it in last pr